### ISSUE TITLE
Next turn street's name occupies a lot of space in landscape navigation

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#2177]
It would be better to display it in a more compact way, near the next direction sign, like some other nav apps are doing. And ideally also make it more recognizable by a driver.

The alternative opinion is to completely remove it because many drivers can't see the small street's name anyway.

Or draw it only if the physical screen dimensions are big enough (like on tables or phablets).

![image](https://user-images.githubusercontent.com/170263/155153413-4384ae74-6886-48e2-ae25-1b6daff3cd5e.png)

**Comment by Donkey-Doug** [comment]
Or use a floating top bar. I suggest adding an option to remove it in settings though.

**Comment by AntonM030481** [comment]
It should be different for vertical and horizontal orientation. E.g.
![image](https://preview.redd.it/jdf9dmzd8i521.jpg?auto=webp&s=06de66e92153e62b0c58409b138e20fae6d0962a)
<img src="https://user-images.githubusercontent.com/85622715/170223920-326a41b9-53fd-48ae-b150-16951388e489.jpeg" width="400" height="auto" />

**Comment by tvld** [comment]
I don't think you need the word "to". If you see a street name, "to" is logical and the arrow makes it clear

**Comment by AntonM030481** [comment]
Agree. This is just reference from Apple Maps.

**Comment by AntonM030481** [comment]
Another example from Waze.
This is good, but distance takes too much space (whole line). So I would keep it under the sign, just like now.

![image](https://user-images.githubusercontent.com/85622715/171575240-7dbf9c24-2359-49ff-a0fa-498e87b10821.png)

**Comment by AntonM030481** [comment]
One of options for horizontal mode is of full info:
- [exit number]: [destination road number] > Destinations - for exits
- [road number]: Road_Name - for turns to street

We can show show short info:
- [exit number]: [road number] - for exits
- nothing - for turns to streets

**Comment by biodranik** [comment]
It would be great to always display full info as Google does.

**Comment by AntonM030481** [comment]
Previous screen was from Waze. This is from Google:
![image](https://user-images.githubusercontent.com/85622715/171989294-bd1ca444-a651-4c9c-b985-6db5877d0df9.png)
